db/model/Variable.ts

```diff
@@ -88,27 +133,14 @@ export async function getVariableData(
         [variableId]
     )
 
-    const dataQuery = db.queryMysql(
-        `
-        SELECT
-            value,
-            year,
-            entities.id AS entityId,
-            entities.name AS entityName,
-            entities.code AS entityCode
-        FROM data_values
-        LEFT JOIN entities ON data_values.entityId = entities.id
-        WHERE data_values.variableId = ?
-        ORDER BY
-            year ASC
-        `,
-        [variableId]
-    )
-
     const row = await variableQuery
-
     if (row === undefined) throw new Error(`Variable ${variableId} not found`)
 
+    const results =
+        row.catalogPath && row.shortName
```

Comment:
Might be nice to assign this and statement to a descriptive variable. As is, I wouldn't know why `catalogPath` and `shortName` mean we use parquet.

Reply:
Neat, fixed.